src/autogen.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-# Copyright [2020] EMBL-European Bioinformatics Institute
+# Copyright [2020-2021] EMBL-European Bioinformatics Institute
 #
```

Comment:
must be 2018-2021 i think, and the same pr for production branch

Reply:
Thanks a lot Mira, will update this file to have the same copyright string as the other files in this repo.